serverlessworkflow/modules/ROOT/pages/index.adoc

```diff
@@ -135,83 +153,68 @@ Learn how to use {serverless_logic_web_tools_name} for creating and managing wor
 [.card]
 --
 [.card-title]
-xref:service-orchestration/orchestration-of-openapi-based-services.adoc[Orchestrating the OpenAPI services]
+xref:service-orchestration/orchestration-of-openapi-based-services.adoc[]
```

Comment:
I would remove this card from the index. This is a "second level" card and I think it does not need to appear here,.

Reply:
I don't think we should remove it. It's a basic/common use case. I'll keep it here.

Reply:
We are talking about the cards here and not the navbar.
I think the cards should remain simple.
We still have OpenAPI card but not the details

Reply:
![Screenshot 2023-06-26 at 15 58 13](https://github.com/kiegroup/kogito-docs/assets/1538000/d83e462a-db4a-4b1f-8742-68ada0773877)

See? This is the main guide.

Reply:
There was 3 openapi cards before ^^ sorry I missed the update. good to me